tests/publiccloud/register_system.pm

```diff
@@ -37,18 +37,19 @@ sub run {
         $args->{my_instance}->retry_ssh_command("sudo SUSEConnect -r $regcode", timeout => 420, retry => 3);
         my $arch = get_var('PUBLIC_CLOUD_ARCH') // "x86_64";
         $arch = "aarch64" if ($arch eq "arm64");
+        my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
         for my $addon (@addons) {
             next if ($addon =~ /^\s+$/);
             record_info $addon, "Going to register '$addon' addon";
             if ($addon =~ /ltss/) {
                 $regcode = get_required_var('SCC_REGCODE_LTSS');
-                ssh_add_suseconnect_product($args->{my_instance}->public_ip, get_addon_fullname($addon), '${VERSION_ID}', $arch, "-r $regcode");
+                ssh_add_suseconnect_product($remote, get_addon_fullname($addon), '${VERSION_ID}', $arch, "-r $regcode");
```

Comment:
just for better understanding can you please elaborate what is going on here ? why before it was fine just to pass IP and now we need to pass "username at public IP " ? I missing how your change related to this

Reply:
Before, we `ssh` as `root` but that is not needed and is actually disabled by default.
We enable `root` access for the ssh tunneled tests where it is required.